lib/foreman_rh_cloud/engine.rb

```diff
@@ -117,6 +117,7 @@ def self.register_scheduled_task(task_class, cronline)
               parent: :insights_menu,
               if: -> { !ForemanRhCloud.with_local_advisor_engine? }
             menu :top_menu, :insights_hits, caption: N_('Recommendations'), url: '/foreman_rh_cloud/insights_cloud', url_hash: { controller: :react, action: :index }, parent: :insights_menu
+            menu :top_menu, :insights_vulnerabilities, caption: N_('Vulnerabilities'), url: '/foreman_rh_cloud/insights_vulnerabilities', url_hash: { controller: :react, action: :index }, parent: :insights_menu
```

Comment:
```suggestion
            menu :top_menu, 
              :insights_vulnerabilities, 
              caption: N_('Vulnerabilities'), 
              url: '/foreman_rh_cloud/insights_vulnerabilities', 
              url_hash: { controller: :react, action: :index }, 
              parent: :insights_menu,
              if: -> { ForemanRhCloud.with_local_advisor_engine? }
```
We don't want to show the page if local engines are not enabled

Reply:
So why don’t we have this condition for the `insights_hits` item as well?

Reply:
You can see hits in both local and connected scenario